tasks/eslint.js

```diff
@@ -9,8 +9,15 @@ module.exports = grunt => {
 			quiet: false,
 			maxWarnings: -1,
 			failOnError: true,
+			extends: false
```

Comment:
This is misleading imo. The documentation clearly says it accepts a `string` or an `array`, but here you define it as a `boolean`. I'd remove this line altogether instead.

Reply:
Sure, I was following the example of `outputFile` which actually defaults to false. Those defaults are not easily accessible and I didn't think it mattered. But the change definitely make sense :+1: